catalyst/data/utils.py

```diff
@@ -0,0 +1,33 @@
+from typing import List, Union
+
+from torch import int as tint, long, short, Tensor
+
+
+def prepare_labels(labels: Union[Tensor, List[int]]) -> List[int]:
```

Comment:
could we rename it to `process_labels`? (we have some internal notation guide :) )